rules/linux/auditd/lnx_auditd_impair_system_power_settings.yml

```diff
@@ -0,0 +1,28 @@
+title: Impair System Power Settings
```

Comment:
Hi,
Masking a service makes it permanently unusable unless we unmask it.
For me more like "Invalidate  System Power Setting"

Reply:
Hi,
Thank you for providing a better name for the rule.